docs/en/connector-v2/source/dingtalk.md

```diff
@@ -0,0 +1,71 @@
+# DingTalk
+
+> DinkTalk source connector
+## Description
+
+A source plugin which use DingTalk
+
+## Key features
+
+- [x] [batch](../../concept/connector-v2-features.md)
+- [ ] [stream](../../concept/connector-v2-features.md)
```

Comment:
It can be seen from the code that the connector supports the stream module. Please check it.